src/main/java/org/sopt36th/seminar/domain/enums/ContractState.java

```diff
@@ -0,0 +1,7 @@
+package org.sopt36th.seminar.domain.enums;
+
+
+public enum ContractState {
+    ACTIVE,
+    INACTIVE,
+}
```

Comment:
1. P5; enum으로 만들 생각한 거 너무 좋다...
2. P5; 와중에 패키지 이름 enums인 것도 조아요
3. P2; 근데 하나 더 생각해본다면 뷰3에서 상태가 정상이니까, 아래처럼 바꾸는 건 어떨까용?
```suggestion
public enum ContractState {
    ACTIVE("정상"),
    INACTIVE("사용 불가")
    
    private final String state;
    
    // 생성자
   ContractState(String state) {this.state = state;}
   
   // getter
   
   // 한국어 이름 받으면 해당 enum 값 반환하는 static 함수
```
대충 써본 거라 문법 틀릴 수도 있어서 느낌만 살짝 봐주세여 ㅎ,,

Reply:
헉..그러게요 이거 진짜 좋은 것 같아요...!!!

Reply:
와 ㄹㅇ 이렇게 하면 훨씬 좋겠네요.. 감사합니다잉!!